app/commands/v2/patch_link_set.rb

```diff
@@ -112,6 +113,19 @@ def send_to_message_queue(content_item)
 
         PublishingAPI.service(:queue_publisher).send_message(payload)
       end
+
+      def format
+        Queries::GetLatest.call(
+          ContentItem.where(content_id: content_id)
+        ).last.try(:format)
```

Comment:
This is an expensive query. Can we get the format from elsewhere? At the least, can we do something like:
```ruby
Queries::GetLatest.call(
  ContentItem.where(content_id: content_id)
).limit(1).pluck(:format)
```
That way we don't need to instantiate ActiveRecord.

Also, I think this should be `schema_name` now, not `format`.